modules/core/src/main/java/com/illposed/osc/OSCParser.java

```diff
@@ -107,7 +108,7 @@ public OSCParser(
 	public static void align(final ByteBuffer input) {
 		final int mod = input.position() % ALIGNMENT_BYTES;
 		final int padding = (ALIGNMENT_BYTES - mod) % ALIGNMENT_BYTES;
-		input.position(input.position() + padding);
+		((Buffer)input).position(input.position() + padding);
```

Comment:
I'm not sure that there is a single place where we could put that explanation. If you'd like, I can go find all the places where I did this cast and add a comment, although I think it would be repetitive and kind of messy.

We should consider what problem we are trying to solve. If we are trying to make sure that people will continue to add this type of cast going forward, then we could solve that problem by adding openjdk11 to the Travis config.